packages/app/lib/src/features/home/presentation/home/post.dart

```diff
@@ -58,6 +57,59 @@ class _PosterInfo extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    // Have post info show how long ago in bar
+    // Show actual date and time of post if you click on it
+    final timeSincePost = post.timestamp.difference(DateTime.now()) +
+        post.timestamp.timeZoneOffset; // Duration values are negative
+    int timeValue;
+    String timePostValue;
+    if (timeSincePost.inDays < -364) {
```

Comment:
When is it less than a negative? This is a bit confusing. Also, you ought to be able to use a switch here, which is nice. You can't usually do that.

Reply:
I just changed how this worked. Its a positive difference now